content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
 
 <TopBlock>
 
-This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality (Preview). Seeding allows you to consistently re-create the same data in your database and can be used to:
+This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality. Seeding allows you to consistently re-create the same data in your database and can be used to:
 
 - Populate your database with data that is required for your application to start - for example, a default language or a default currency.
-- Provide basic data for validating and using your application in a development environment. This is particulary useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
-
-Watch the following video for an introduction to the `db seed` command:
-
-<div class="videoWrapper">
-
-<iframe
-  width="560"
-  height="315"
-  src="https://www.youtube.com/embed/2LwTUIqjbPo"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
-
-</div>
+- Provide basic data for validating and using your application in a development environment. This is particularly useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
 
 </TopBlock>
 
-## Requirements for seeding with TypeScript or JavaScript
-
-Prisma's integrated seeding functionality expects the following:
-
-<!-- prettier-ignore-start -->
+## Configuring your seed script
```

Comment:
Looking at the preview, this still feels to specific. This is more about "enabling seeing in Prisma" or "Configuring the seeding command for Prisma to use" or similar.

Reply:
SEO title FTW: How to seed your database in Prisma